appengine/flexible_ubuntu/storage/requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest==7.0.1
+google-cloud-storage==2.1.0; python_version > '3.6'
```

Comment:
Since you aren't allowing versions less than 3.7 anyway, the "; python_version > '3.6." isn't really needed. There's no alternate requirement for lower Python version.